static/product-designer-portfolio/index.html

```diff
@@ -0,0 +1,285 @@
+            >
+              <p class="text-center text-lg font-semibold text-white">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              </p>
+            </div>
+          </div>
+          <div class="group relative m-2 max-h-[600px]">
+            <img
+              src="https://images.unsplash.com/photo-1491637639811-60e2756cc1c7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MjR8fHByb2R1Y3QlMjBkZXNpbmdlcnxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60"
+              alt="description"
+              class="h-full w-full object-cover"
+            />
+            <div
+              class="absolute left-1/2 top-1/2 flex h-full w-full -translate-x-1/2 -translate-y-1/2 items-center justify-center overflow-hidden bg-[black]/20 opacity-0 duration-500 group-hover:w-full group-hover:opacity-100"
+            >
+              <p class="text-center text-lg font-semibold text-white">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              </p>
+            </div>
+          </div>
+          <div class="group relative m-2 max-h-[600px]">
+            <img
+              src="https://images.unsplash.com/photo-1524638067-feba7e8ed70f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8OXx8cHJvZHVjdCUyMGRlc2luZ2VyfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60"
+              alt="description"
+              class="h-full w-full object-cover"
+            />
+            <div
+              class="absolute left-1/2 top-1/2 flex h-full w-full -translate-x-1/2 -translate-y-1/2 items-center justify-center overflow-hidden bg-[black]/20 opacity-0 duration-500 group-hover:w-full group-hover:opacity-100"
+            >
+              <p class="text-center text-lg font-semibold text-white">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              </p>
+            </div>
+          </div>
+          <div class="group relative m-2 max-h-[600px]">
+            <img
+              src="https://images.unsplash.com/photo-1549049950-48d5887197a0?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1528&q=80"
+              alt="description"
+              class="h-full w-full object-cover"
+            />
+            <div
+              class="absolute left-1/2 top-1/2 flex h-full w-full -translate-x-1/2 -translate-y-1/2 items-center justify-center overflow-hidden bg-[black]/20 opacity-0 duration-500 group-hover:w-full group-hover:opacity-100"
+            >
+              <p class="text-center text-lg font-semibold text-white">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- ==== End Featured Work ====  -->
+
+    <!-- ==== Start Visual Explorations ====  -->
+    <section class="my-16">
+      <div class="container">
+        <h1 class="text-light-white mb-8 text-center text-3xl">
```

Comment:
Using more than one <h1> is allowed by the HTML specification, but is not considered a best practice. Using only one <h1> is beneficial for screenreader users.

So in this case the hero text could be probably h1, and the rest should be lower.

Reply:
I'm using a section tag, and the section should have at least 1 heading. should I replace the section with div?

Reply:
What I meant is that you should use h2, h3, h4... other appropriate heading elements. As one page should usually have only one h1.